src/schema/rogue.js

```diff
@@ -577,9 +585,11 @@ const typeDefs = gql`
       count: Int = 20
     ): [Post]
     schoolActionStats(
-      "The School ID to filter school action stats by."
+      "The School ID to filter action stats by."
       schoolId: String
-      "The Action ID to filter school action stats by."
+      "The school location to filter action stats by."
```

Comment:
```suggestion
      "The ISO-3166-2 school location to filter action stats by (e.g. US-NY)."
```
It might make sense to note that this is an [ISO-3166-2](https://en.wikipedia.org/wiki/ISO_3166-2) location (so it's clear what kind of string to provide).